pkg/sink/sink_test.go

```diff
@@ -188,13 +240,21 @@ func Test_createResource(t *testing.T) {
 	tests := []struct {
 		name        string
 		resource    json.RawMessage
+		namespace   string
 		wantURLPath string
 	}{
 		{
 			name:        "PipelineResource",
 			resource:    json.RawMessage(pipelineResourceBytes),
+			namespace:   pipelineResource.Namespace,
```

Comment:
nit: If I understand correctly, this namespace should probably be a different namespace than the one in the PipelineResource, because this namespace should be the namespace of the EventListener.

Can you also please add a test case for a resource created without its `metadata.namespace` field?